torch/distributed/checkpoint/planner_helpers.py

```diff
@@ -189,6 +192,8 @@ def _create_write_items_for_dtensor(fqn: str, tensor: DTensor) -> WriteItem:
             size=tensor.size(),
         ),
     )
+    logger.warning(f"creating {write_item=} for {fqn=} with {offsets=} and {sizes=}")
```

Comment:
f-string will complain here

Reply:
Oh this is a not-for-landing change to debug something